e2eTest/src/main/java/com/google/android/ground/e2etest/SurveyRunnerTest.kt

```diff
@@ -65,25 +83,39 @@ class SurveyRunnerTest : AutomatorRunner {
 
     // Wait for the app to appear.
     if (!device.wait(Until.hasObject(By.pkg(GROUND_PACKAGE).depth(0)), LONG_TIMEOUT)) {
+      takeScreenshot("${javaClass.simpleName}_${nameRule.methodName}")
```

Comment:
`${javaClass.simpleName}_${nameRule.methodName}` is used a few times.. you might be able to extract it into to an `inline` helper, eg `this.getScreenshotName()`.

Reply:
Done.